wger/manager/migrations/0009_merge.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9 on 2019-06-20 09:31
+# Generated by Django 1.9 on 2019-06-18 12:58
```

Comment:
@Manorlds-Eaglespark you can delete this file and thereafter rebase with `develop`. We need to maintain the one that has already been delivered.

Reply:
Resolved. Thanks @Rhytah